configs/login-identity-providers-ldap.xml

```diff
@@ -83,8 +83,8 @@
         <property name="Read Timeout">10 secs</property>
         <property name="Url">{{.Values.auth.ldap.host}}</property>
         <property name="User Search Base">{{.Values.auth.ldap.searchBase}}</property>
-        <property name="User Search Filter">(cn={0})</property>
+        <property name="User Search Filter">{{.Values.auth.ldap.searchFilter}}</property>
```

Comment:
I think this incorrect and should be `({{.Values.auth.ldap.userIdentityAttribute}}={0})` See: [NiFi Docs of login-identity-providers.xml section](https://nifi.apache.org/docs/nifi-docs/html/administration-guide.html#:~:text=DC%3Dcom).-,User%20Search%20Filter,sAMAccountName%3D%7B0%7D).%20The%20user%20specified%20name%20is%20inserted%20into%20%27%7B0%7D%27.,-Identity%20Strategy).